Controller/Adminhtml/Config/Disconnect.php

```diff
@@ -73,15 +81,36 @@ public function __construct(
      */
     public function execute()
     {
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_APIKEY, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_EMAIL, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_CONNECTED, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_ENABLED, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_BACKUP, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_AUTH, 'default', 0);
-        $this->resourceConfig->deleteConfig(TaxjarConfig::TAXJAR_TRANSACTION_SYNC, 'default', 0);
-        $this->reinitableConfig->reinit();
+        // Erase config values with the "default" scope
```

Comment:
```
Exception #0 (Exception): Warning: Missing argument 2 for Magento\Config\Model\ResourceModel\Config::deleteConfig(),
called in /app/code/Taxjar/SalesTax/Controller/Adminhtml/Config/Disconnect.php on line 85
and defined in /vendor/magento/module-config/Model/ResourceModel/Config.php on line 73
```

Default values for `$scope` and `$scopeId` were not introduced until [v2.2.5](https://github.com/magento/magento2/blob/2.2.5/app/code/Magento/Config/Model/ResourceModel/Config.php#L75). So we'll need to keep `, 'default', 0`